### DISCUSSION TITLE
Criando Apps Web com Angular e UiKit - erro clonar UiKit

### DISCUSSION THREAD

**Discussion by gilberto-moral** [balta-io/forum#363] [Tira Dúvidas] [answered]
Erro na permissão da instalação (clonar) Uikit:
![image](https://github.com/balta-io/forum/assets/1138725/e3537632-75d9-4737-8419-c0be8e89ce9a)

**Comment by andrebaltieri** [comment, marked as answer]
@gilberto-moral boa noite, tentou rodar com SUDO?

**Reply by gilberto-moral** [reply]
Na verdade peguei o código fonte do Github, fiz assim, única maneira que consegui seguir.

**Reply by andrebaltieri** [reply]
Me parece um erro no seu WSL que impede do repositório ser clonado... Por isso recomendei usar o SUDO!
